integ-test-promote.pipeline.yml

```
@@ -207,6 +207,46 @@ stages:
               grep -v FirstTest test-results/matlab/selectbytag.xml
               grep -v simpleTest test-results/matlab/selectbytag.xml
             displayName: Verify test filtered by tag name
+          - bash: |
+              echo 'diary console.log' >> startup.m
+            displayName: Setup diary for logging
+          - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABTests.RunMATLABTests@0
+            displayName: Run MATLAB tests with strict checks
+            inputs:
+              strict: true
+              sourceFolder: src
+          - bash: |
+              set -e
+              grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
+            displayName: Verify tests ran with strict checks
```

```suggestion
          - bash: |
              set -e
              grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
              rm console.log
            displayName: Verify tests ran with strict checks
```
It might be good to clean up the `console.log` file in between each of these checks to make sure it's only capturing the information from the most recent run. I'm not sure if it replaces or just appends each time.